src/styles/_stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  line-height: 16px;
+  width: 100%;
+  margin-top: 16px;
+
+  &__list {
+    @include center;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    justify-content: space-between;
```

Comment:
The `justify-content` property is used here, but it will only have an effect if the `.stars__star` element is a flex container. Ensure that the `.stars__star` class has `display: flex;` applied if you intend to use `justify-content`.